static/app/components/forms/model.tsx

```diff
@@ -10,7 +10,7 @@ import {defined} from 'sentry/utils';
 type Snapshot = Map<string, FieldValue>;
 type SaveSnapshot = (() => number) | null;
 
-export type FieldValue = string | number | boolean | undefined; // is undefined valid here?
+export type FieldValue = string | number | boolean | undefined | Array<string | number>; // is undefined valid here?
```

Comment:
Also not sure I understand the use case for the value of the field being an array. Are we introducing a new field type?